subsys/bluetooth/host/att.c

```diff
@@ -1204,9 +1205,13 @@ static u8_t write_cb(const struct bt_gatt_attr *attr, void *user_data)
 		return BT_GATT_ITER_STOP;
 	}
 
-	/* Read attribute value and store in the buffer */
+	if (!data->op) {
+		flags |= BT_GATT_WRITE_FLAG_CMD;
```

Comment:
How come data->op is 0 for BT_ATT_OP_WRITE_CMD and not, well, BT_ATT_OP_WRITE_CMD? Seems a bit strange. If there's a good reason for it there should at least be a short code comment explaining that 0 means BT_ATT_OP_WRITE_CMD.

Reply:
The intention was only to set it for requests which requires responses to avoid having extra checks for what type of request later on. Both write cmd and signed write cmd don't set a request opcode.